frontend/packages/employee-frontend/src/assets/i18n/fi.ts

```diff
@@ -1237,6 +1240,32 @@ export const fi = {
       save: 'Tallenna muutokset'
     }
   },
+  valueDecisions: {
+    table: {
+      title: 'Arvopäätökset',
+      rowCount: (count: number) =>
+        count > 0 ? `Hakutuloksia: ${count}` : 'Ei hakutuloksia',
+      head: 'Päämies',
+      children: 'Lapset',
+      validity: 'Arvopäätös voimassa',
+      number: 'Numero',
+      status: 'Tila',
+      createdAt: 'Luotu',
+      sentAt: 'Lähetetty',
+      paging: 'Sivu'
+    },
+    buttons: {
+      checked: (count: number) =>
```

Comment:
No need to fix this, mut a simpler solution: checked: (count: number) => `Maksupäätöksiä valittu ${count}` ?

Reply:
Yeah, but then it's not as fluent 😌